src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewMonth.razor.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Globalization;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewMonth : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public byte? Month { get; set; }
+
+    [Parameter]
+    public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+
+    public Dictionary<byte, string?>? MonthSelectionList { get; set; } = new Dictionary<byte, string?>();
+
+    public string DisplayMonth
+    {
+        get
+        {
+            if (MonthSelectionList == null
```

Comment:
is there a case where MonthSelectionList would be null?

Reply:
resolved on commit